swarm/api/routes/issue_routes.py

```
@@ -0,0 +1,258 @@
+        return IssueIngestionResponse(
+            run_id=run_id,
+            status="created",
+            issue_snapshot_path=str(snapshot_path.relative_to(run_dir)),
+            autopilot_started=autopilot_started,
+            events_url=f"/api/runs/{run_id}/events",
+            created_at=now.isoformat(),
+        )
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)